README.md

```diff
@@ -49,6 +49,9 @@ If you want `ember server` to start the Hoodie Server for you, you have to confi
 // config/environment.js
 if (environment === 'development') {
   ENV.hoodie = {
+    client: {
+      url: 'http://127.0.0.1:4201'
+    },
```

Comment:
hmm... you're right... Maybe I got another error and rushed into this solution...

anyway, maybe it should help when deploying in production?! just wondering...